api/agent/agent.go

```diff
@@ -609,6 +636,18 @@ func (s *hotSlot) exec(ctx context.Context, call *call) error {
 	}
 }
 
+// nonZeroMin returns zero only if both parameters are zero, otherwise it
+// returns the minimum non-zero number of the two.
```

Comment:
For CPU, zero value in call means no limit & skip. In this case, nonZeroMin() should not pick up MaxCPU limit, since it'll accidentally set max CPU to no-limit calls.